src/store/reducers/customerReducer.jsx

```diff
@@ -0,0 +1,79 @@
+import { createSlice } from '@reduxjs/toolkit';
+import customer from '../../features/client/mocks/customer';
+
+const initialState = customer;
+
+export const customerSlice = createSlice({
+  name: 'customer',
+  initialState,
+  reducers: {
+    edit: (state, action) => {
```

Comment:
**update** instead of edit, since edit is usually referred to edit page that **updates** data